android/src/main/java/com/reactnativepagerview/FragmentAdapter.kt

```diff
@@ -9,6 +9,8 @@ import java.util.*
 
 class FragmentAdapter(fragmentActivity: FragmentActivity) : FragmentStateAdapter(fragmentActivity) {
   private val childrenViews: MutableList<View> = ArrayList()
+  var initialPageinited = false
```

Comment:
I needed a local state to avoid calling setInitialPage every time a component render.
Can it be put in a better place?